backend/src/appointment/routes/auth.py

```diff
@@ -237,6 +241,13 @@ def me(
         timezone=subscriber.timezone, avatar_url=subscriber.avatar_url
     )
 
+
+@router.post("/permission-check")
+def permission_check(_: Subscriber = Depends(get_admin_subscriber)):
+    """Checks if they have admin permissions"""
+    return True
```

Comment:
The check is done in get_admin_subscriber, and will error out. FastAPI has a weird relationship with using DI over middleware...